frontend/src/app/login/page.tsx

```diff
@@ -0,0 +1,85 @@
+"use client";
+
+import { useState } from "react";
+import { useRouter } from "next/navigation";
+import Navbar from "@/components/navbar";
+
+const LoginPage = () => {
+  const [form, setForm] = useState({ email: "", password: "" });
+  const [error, setError] = useState("");
+  const router = useRouter();
+
+  const handleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setForm({
+      ...form,
+      [e.target.name]: e.target.value,
+    });
+  };
+
+  const handleSubmit = async (e: React.FormEvent) => {
+    e.preventDefault();
+    try {
+      const res = await fetch(`${process.env.NEXT_PUBLIC_API_URL}auth/login`, {
```

Comment:
remove the `/` from .env and add here for better readability
```suggestion
      const res = await fetch(`${process.env.NEXT_PUBLIC_API_URL}/auth/login`, {
```